src/sfctl/custom_app.py

```diff
@@ -50,6 +77,15 @@ def path_from_imagestore_string(imagestore_connstr):
         return conn_str_list[1]
     return False
 
+def get_job_count():
+    """
+    Test-mockable wrapper for returning cpu count.
+    """
+    jobcount = cpu_count()
```

Comment:
I believe this can throw also, should put it in a try I believe.

Reply:
ok